example/workerFile.js

```diff
@@ -44,5 +44,18 @@ const workerFunction = () => {
     hermes.on("time-transfer", (date, bigFile) => {
         console.log(`${new Date().getTime() - date.getTime()}ms`);
     });
+
+    hermes.on("canvas2D", (canvas) => {
+        const ctx = canvas.getContext("2d");
```

Comment:
Add warning for Firefox

Reply:
why does FF need a warning ?

Reply:
To Avoid an error,
Firefox have bug for context2dRendering in worker ! (https://bugzilla.mozilla.org/show_bug.cgi?id=801176)

But its work if we use WebGL Rendering ! 

I think add error for other unsupported browser ! 
This is an experimental feature .

Reply:
okay I didn't know, thanks !

Reply:
Yes, that's why i wanted to put an log ;)

Reply:
@robincartier I have add log in example/script.js i want your opinion !